overview.md

```diff
@@ -6,18 +6,19 @@ The Endgame Plan is a proposal to overhaul and improve the governance and tokeno
 
 The Endgame Plan was [introduced](https://forum.makerdao.com/t/decentralized-voter-committee-on-tuesday-5pm-cest-all-mkr-holders-welcome/15166) by Rune Christensen, the co-founder of Maker. After a series of initial discussions, Maker Governance accepted the [MIP set](https://vote.makerdao.com/polling/QmTmS5Nf) ratifying this plan on 24 October 2022. 
 
-The Endgame Plan remains in active development, led by Rune Christensen. The latest complete and published description of the Endgame Plan is available [here](https://forum.makerdao.com/t/endgame-plan-v3-complete-overview/17427). Note that some parts are now out of date.  
+The Endgame Plan remains in active development, led by Rune Christensen. The latest complete and published description of the Endgame Plan is available [here](https://forum.makerdao.com/t/endgame-plan-v3-complete-overview/17427). The Endgame Constitution, covering the  proposed by Christensen is available [here](https://forum.makerdao.com/t/mip101-the-maker-constitution/19621). Note that some parts are now out of date.  
 
 
 {% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
 
 ## Goals
 
 The Endgame Plan has many goals which come together to create the Endgame State. A list of these goals is below:
+- Providing access to an Unbiased World Currency in the form of the decentralized stablecoin Dai.
+- Ensure that Maker has a meaningful charitable impact through financial success.
```

Comment:
Not sure this quite makes sense. financial success doesn't exactly equate to charitable impact. Probably just end at charitable impact?